test/User-test.js

```diff
@@ -79,7 +81,44 @@ let user;
       dailyStepGoal: 10000,
       friends: [16, 4, 8],
     });
-    expect(fakeUser.getAvgSleepDataPerDay(sleepTestData, "sleepQuality")).to.equal(0)
-    expect(fakeUser.getAvgSleepDataPerDay(sleepTestData, "hoursSlept")).to.equal(0)
-  })
+    expect(
+      fakeUser.getAvgSleepDataPerDay(sleepTestData, "sleepQuality")
+    ).to.equal(0);
+    expect(
+      fakeUser.getAvgSleepDataPerDay(sleepTestData, "hoursSlept")
+    ).to.equal(0);
+  });
+  it("should have a method to calculate the hours slept per day over a week for a user and quality of sleep ", () => {
+    let sleeptestData;
```

Comment:
This variable is not needed or used anywhere I don't believe.

Reply:
okay, I wasn't sure.

Reply:
done